packaging/functions.sh

```diff
@@ -66,7 +66,7 @@ install_assemblies_mono() {
 	fi
 
 	# Managed Dependencies
-	for LIB in ICSharpCode.SharpZipLib.dll FuzzyLogicLibrary.dll Open.Nat.dll BeaconLib.dll DiscordRPC.dll Newtonsoft.Json.dll SDL2-CS.dll OpenAL-CS.Core.dll Eluant.dll; do
+	for LIB in ICSharpCode.SharpZipLib.dll FuzzyLogicLibrary.dll Open.Nat.dll BeaconLib.dll DiscordRPC.dll Newtonsoft.Json.dll SDL2-CS.dll OpenAL-CS.Core.dll Eluant.dll System.Net.Http.dll; do
```

Comment:
Ugh... this doesn't actually work 😞  https://github.com/pchote/OpenRA/actions/runs/435506045

This leaves us with three options:
1. go back to one big PR that switches everything to net 5 in one go
2. update my mono support bundles for macOS and Linux to add this (and change the makefile back to WHITELISTED_CORE_ASSEMBLIES)
3. stay with `WebClient` until we've switched our release packages to Core (removing the dependency on 2)

1 isn't reasonable, and i'd rather not go to have to do 2 so close to finally getting rid of them, so can we please change this to just the `Task.Run` switch?